src/algebra/algebra/operations.lean

```diff
@@ -68,6 +75,8 @@ begin
   simp only [mem_one, mem_span_singleton, algebra.smul_def, mul_one]
 end
 
+theorem one_eq_span_one_set : (1 : submodule R A) = span R 1 := one_eq_span
```

Comment:
That sounds like it should be a simp lemma the other way around.

Reply:
This is to match `mul_eq_span_mul_set` which already exists.